deps/spidershim/src/v8trycatch.cc

```diff
@@ -41,7 +41,7 @@ struct TryCatch::Impl {
     // TODO: Propagate exceptions to the caller
     //       https://github.com/mozilla/spidernode/issues/51
   }
-  Isolate* Isolate() const { return isolate_; }
+  Isolate* Iso() const { return isolate_; }
```

Comment:
On Wed, May 04, 2016 at 10:39:55PM -0700, Ehsan Akhgari wrote:
> In deps/spidershim/src/v8trycatch.cc:
> 
> > @@ -41,7 +41,7 @@ struct TryCatch::Impl {
> >      // TODO: Propagate exceptions to the caller
> >      //       https://github.com/mozilla/spidernode/issues/51
> >    }
> > -  Isolate* Isolate() const { return isolate_; }
> > +  Isolate* Iso() const { return isolate_; }
> 
> Can you instead use class Isolate* Isolate() const ... instead? That should
> help fix this error without needing to rename Isolate(), which is ugly. We
> prefix names with class like this elsewhere in the code too.

No, that doesn't work because the method is still changing the meaning
of Isolate in the class scope from the type to the method, which aiui is
the problem.  Its also what Botond suggested, and I'm not going to try
and question him on C++ name lookup rules ;)